src/main/java/net/william278/velocitab/config/Formatter.java

```diff
@@ -79,6 +79,11 @@ public Component format(@NotNull String text, @NotNull TabPlayer player, @NotNul
         return formatter.apply(text, player, plugin);
     }
 
+    @NotNull
+    public String format(@NotNull String text) {
```

Comment:
```suggestion
    public String formatLegacySymbols(@NotNull String text) {
```
I think this method name's a little more sensible